docs/reference/object-detection-2d-nanodet.md

```diff
@@ -0,0 +1,413 @@
+| Nanodet-m           {416}    | 47.3     | 8.34  | 13.15 |
+| Nanodet-m 1.5x      {416}    | 45.62    | 8.43  | 13.2  |
+| Nanodet-plue m      {320}    | 41.9     | 7.45  | 12.01 |
+| Nanodet-plue m 1.5x {320}    | 39.63    | 7.66  | 12.21 |
+| Nanodet-plue m      {416}    | 40.16    | 7.24  | 11.58 |
+| Nanodet-plue m 1.5x {416}    | 38.94    | 7.37  | 11.52 |
+
+For JIT optimization inference.
+
+| Method              {intput} | RTX 2070 | TX2   | NX    |
+|------------------------------|----------|-------|-------|
+| Efficient Lite0     {320}    | 69.06    | 12.94 | 17.78 |
+| Efficient Lite1     {416}    | 62.94    | 9.27  | 12.94 |
+| Efficient Lite2     {512}    | 65.46    | 7.46  | 10.32 |
+| RepVGG A0           {416}    | 41.44    | 11.16 | 14.89 |
+| Nanodet-g           {416}    | 76.3     | 12.94 | 20.52 |
+| Nanodet-m           {320}    | 75.66    | 12.22 | 20.67 |
+| Nanodet-m 0.5x      {320}    | 65.71    | 11.31 | 17.68 |
+| Nanodet-m 1.5x      {320}    | 66.23    | 12.46 | 19.99 |
+| Nanodet-m           {416}    | 79.91    | 12.08 | 19.28 |
+| Nanodet-m 1.5x      {416}    | 69.44    | 12.3  | 18.6  |
+| Nanodet-plue m      {320}    | 67.82    | 11.19 | 18.85 |
+| Nanodet-plue m 1.5x {320}    | 64.12    | 11.57 | 18.26 |
+| Nanodet-plue m      {416}    | 64.74    | 11.22 | 17.57 |
+| Nanodet-plue m 1.5x {416}    | 56.77    | 10.39 | 14.81 |
+
+For ONNX optimization inference.
+
+In this case, the forward pass is performed in ONNX.
+The pre-processing steps were implemented in PyTorch.
+Results show that the performance on ONNX varies significantly among different architectures, with some achieving good performance while others performing poorly.
+Additionally, it was observed that the performance of ONNX on a TX2 device was generally good, although it was observed to have occasional spikes of long run times that made it difficult to accurately measure.
+Overall, the TX2 device demonstrated good performance with ONNX.
+
+| Method              {intput} | RTX 2070  | TX2 | NX     |
+|------------------------------|-----------|-----|--------|
+| Efficient Lite0     {320}    | 33.12     |     | 34.03  |
+| Efficient Lite1     {416}    | 16.78     |     | 17.35  |
+| Efficient Lite2     {512}    | 10.35     |     | 12.14  |
+| RepVGG A0           {416}    | 27.89     |     | 51.74  |
+| Nanodet-g           {416}    | 103.22    |     | 87.40  |
+| Nanodet-m           {320}    | 98.73     |     | 122.26 |
+| Nanodet-m 0.5x      {320}    | 144.46    |     | 208.19 |
+| Nanodet-m 1.5x      {320}    | 75.82     |     | 75.40  |
+| Nanodet-m           {416}    | 73.09     |     | 72.78  |
+| Nanodet-m 1.5x      {416}    | 51.30     |     | 51.78  |
+| Nanodet-plue m      {320}    | 51.39     |     | 50.67  |
+| Nanodet-plue m 1.5x {320}    | 39.65     |     | 40.62  |
+| Nanodet-plue m      {416}    | 39.17     |     | 36.98  |
+| Nanodet-plue m 1.5x {416}    | 28.55     |     | 27.20  |
```

Comment:
As I said previously in the .md file I noticed that the measurements aren't very consistent and have a lot of outliers. I decided not to include them in the file, but I wanted to check with you if you still think it would be a good idea to post them despite that. What do you think?

Maybe It whould be better to change the lines 375 and 376 so they say that the measures are not very consistent and have a lot of outliers.

If that is ok for now I will check in the future if I can make them more concistent in the embededs and update the .md

Reply:
I think that it looks bad to have the column empty. 

I suggest to either keep the explanation with maybe adding something like "..that made it difficult to accurately measure, and as such the measurements are not included in the table below." on line 375 and remove the column entirely, or just add the measurements in the table and keep the disclaimer as it is. I think that adding the measurements would be preferable.

Reply:
@ManosMpampis just noticed another typo in all tables on the first column its `Nanodet-plue`